src/nrtk_explorer/library/multiprocess_predictor.py

```diff
@@ -0,0 +1,149 @@
+import multiprocessing
+import asyncio
+import signal
+import threading
+import logging
+import queue
+import uuid
+from .predictor import Predictor
+
+
+def _child_worker(request_queue, result_queue, model_name, force_cpu):
+    signal.signal(signal.SIGINT, signal.SIG_IGN)  # Ignore Ctrl+C in child
+    logger = logging.getLogger(__name__)
+    predictor = Predictor(model_name=model_name, force_cpu=force_cpu)
+
+    while True:
+        try:
+            msg = request_queue.get()
+        except (EOFError, KeyboardInterrupt):
+            logger.debug("Worker: Exiting on interrupt or queue EOF.")
+            break
+        if msg is None:  # Exit signal
+            logger.debug("Worker: Received EXIT command. Shutting down.")
+            break
+
+        command = msg["command"]
+        req_id = msg["req_id"]
+        payload = msg.get("payload", {})
+        logger.debug(f"Worker: Received {command} with ID {req_id}")
+
+        if command == "SET_MODEL":
```

Comment:
I think we should place these command strings in some constant/enum, since we refer to them in different parts of the application.

Reply:
Sounds good.  Let me do that in another PR right now, as I've made a mess of the history and modifed this file in the following PRs.